src/isPalindrome.js

```diff
@@ -19,7 +19,13 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  if (str.length === 0) {
+    return true;
+  }
+  const strOnlyLetter = str.toLowerCase().match(/\w+/gi).join('');
+  const strOnlyLetterReverse
+        = str.toLowerCase().match(/\w/gi).reverse().join('');
```

Comment:
use `strOnlyLetter ` here
```suggestion
        = strOnlyLetter.split('').reverse().join('');
```